### PR TITLE
CIV-6791 st of means response._Snyk

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link ###

https://[tools.hmcts.net](https://tools.hmcts.net/jira/browse/CIV-6791)/jira/browse/CIV-6791

### Change description ###

Fix bug about directionsQuestionnaireSigned in the check and send view for response and claim journey

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/routes/features/response/checkAnswersController.ts [hunk 1]
```diff
@@ -46,8 +42,9 @@ checkAnswersController.get(RESPONSE_CHECK_ANSWERS_URL,
 
 checkAnswersController.post(RESPONSE_CHECK_ANSWERS_URL, async (req: Request, res: Response, next: NextFunction) => {
   try {
-    const isFullAmountRejected = (req.body?.isFullAmountRejected === 'true');
-    const form = new GenericForm((req.body.type === 'qualified')
+    console.log('TEST:', req.body);
```
Comment (GorilaNaranja) [inline src/main/routes/features/response/checkAnswersController.ts:45]: delete log

Comment (neeta-hmcts) [inline src/main/routes/features/response/checkAnswersController.ts:45]: can be removed.

File: src/main/views/macro/statement-of-truth.njk [hunk 2]
```diff
@@ -1,37 +1,27 @@
...
-  typeOfStatement = t('COMMON.STATEMENT_OF_TRUTH.STATEMENT_OF_WORK'),
-  isFullAmountRejected = false
+  labelHtml
   ) %}
-
   <legend><h2 class="govuk-heading-m">{{ t('COMMON.STATEMENT_OF_TRUTH.TITLE') }}</h2></legend>
-
-  {% if statementOfTruthType == SignatureType.CLAIM or statementOfTruthType == SignatureType.RESPONSE %}
-    <p class="govuk-body">{{ t('COMMON.STATEMENT_OF_TRUTH.INFORMATION_ON_PAGE', { type: statementOfTruthType }) }}</p>
+  {% if form.model.type == SignatureType.RESPONSE or form.model.type == SignatureType.BASIC %}
```
Comment (GorilaNaranja) [inline src/main/views/macro/statement-of-truth.njk:10]: use triple ===

File: src/main/routes/features/response/checkAnswersController.ts (hunk 1 above)
Comment (laarmada) [reply src/main/routes/features/response/checkAnswersController.ts]: push it

Comment (laarmada) [reply src/main/routes/features/response/checkAnswersController.ts]: done

File: src/main/views/macro/statement-of-truth.njk (hunk 2 above)
Comment (laarmada) [reply src/main/views/macro/statement-of-truth.njk]: done

**Review by GorilaNaranja — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]

**Pull request merged**